prompts/csharp-mstest.prompt.md

```diff
@@ -36,7 +36,7 @@ Your goal is to help me write effective unit tests with MSTest, covering both st
 
 ## Data-Driven Tests
 
-- Use `[DataTestMethod]` combined with data source attributes
+- Use `[TestMethod]` combined with data source attributes
```

Comment:
GitHub isn't allowing me to add a comment on the specific line. But on line 43, there is a mention of `CsvDataSource`. I'm not aware of such attribute 😄 
We only have `DataSourceAttribute` which can be set to use Csv. A sample for using Csv is https://github.com/microsoft/testfx/blob/b1edce21fa74ac0444f92355d230a498196a3382/test/IntegrationTests/MSTest.Acceptance.IntegrationTests/DataSourceTests.cs, which is a bit involved.

Reply:
Probably worth removing it then

Reply:
yeah as they are outside the diff commenting isn't supported.

Let's get that fixed and then I can merge these changes.